stdlib/private/StdlibUnittest/RaceTest.swift

```diff
@@ -55,6 +55,8 @@ func autoreleasepool(invoking code: () -> Void) {
 }
 #endif
 
+// FIXME: On FreeBSD, code in this file doesn't compile
```

Comment:
I think this is fine for now, but I'd really like to avoid this kind of changes. Are you taking care of this file in a separate review? Also, I think it would be good to explain why it doesn't build (in a comment).